core/renderers/zelos/drawer.js

```diff
@@ -13,39 +13,40 @@
 goog.module('Blockly.zelos.Drawer');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering');
-goog.require('Blockly.blockRendering.Drawer');
-goog.require('Blockly.utils.object');
-goog.require('Blockly.utils.svgPaths');
-goog.require('Blockly.zelos.RenderInfo');
-
-goog.requireType('Blockly.blockRendering.Row');
-goog.requireType('Blockly.BlockSvg');
-goog.requireType('Blockly.zelos.PathObject');
+const BaseDrawer = goog.require('Blockly.blockRendering.Drawer');
+/* eslint-disable-next-line no-unused-vars */
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+/* eslint-disable-next-line no-unused-vars */
+const PathObject = goog.requireType('Blockly.zelos.PathObject');
+const RenderInfo = goog.require('Blockly.zelos.RenderInfo');
```

Comment:
Looks like this should be a `requireType` (with `eslint-disable` for the unused var warning).